languages/java/exercises/concept/dates/.docs/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Dates
```

Comment:
I found it slightly odd to find this file in here, as superficially people might think there is a dates exercise on account of the directory being there. I'd remove this file and the dates directory. Same goes for the other two exercise directories.